cypress/support/index.js

```diff
@@ -70,3 +70,8 @@ Cypress.Commands.add("iFrame", (selector) => {
 Cypress.Screenshot.defaults({ screenshotOnRunFailure: DEBUG_FLAG });
 
 export default DEBUG_FLAG;
+
+// replace the moment.js because of deprecation
+const dayjs = require("dayjs");
+
+Cypress.dayjs = dayjs;
```

Comment:
Weird that Cypress includes `dayjs` by default but you have to manually require it in like this.

Reply:
yes,  it is :) but they are doing it the same way [here](https://github.com/cypress-io/cypress-example-recipes/blob/f4441ecf2f689f823b1f78b2b884f4164eb0dbe9/examples/blogs__dayjs/cypress/integration/spec.js)